Martinh/llm resources page (#338)
wormhole-foundation/wormhole-docs: learn/ai-resources.md

---
title: AI Resources
description: Download LLM-optimized files of the Wormhole documentation, including full content and category-specific resources for AI agents.
---

# AI Resources
Wormhole provides `.txt` files containing the documentation content and navigation structure, optimized for use with large language models (LLMs) and AI tools. These resources help build AI assistants, power code search, or enable custom tooling trained on Wormhole’s docs.

## Download LLM Files

| Category       | Description       | File  | Actions  |
|----------------|-------------------|-------|----------|
| Index         | Navigation index of all Wormhole documentation pages | `llms.txt`         | [:octicons-copy-16:](){ .llms action="copy" data-value="llms.txt" } [:octicons-download-16:](/docs/llms-files/llms.txt){ download="llms.txt" }                           |

> | Full Documentation  | Full content of all documentation pages | `llms-full.txt`         | [:octicons-copy-16:](){ .llms action="copy" data-value="llms-full.txt" } [:octicons-download-16:](/docs/llms-files/llms-full.txt){ download="llms-full.txt" }                           |

Reviewer: ```suggestion
| Full Documentation  | Full content of all documentation pages | `llms-full.txt`         | [:octicons-copy-16:](){ .llms action="copy" data-value="llms-full.txt" } [:octicons-download-16:](/docs/llms-full.txt){ download="llms-full.txt" }                           |
```
This file is in the root, not in the folder with the categorized files

| Connect        | Setup, features, and configuration details for integrating the Connect widget into your dApp | `llms-connect.txt`        | [:octicons-copy-16:](){ .llms action="copy" data-value="llms-connect.txt" } [:octicons-download-16:](/docs/llms-files/llms-connect.txt){ download="llms-connect.txt" }                        |
| Token Bridge   | Architecture overview, transfer flows, and smart contract methods for cross-chain token transfers using the Token Bridge | `llms-token-bridge.txt`   | [:octicons-copy-16:](){ .llms action="copy" data-value="llms-token-bridge.txt" }   [:octicons-download-16:](/docs/llms-files/llms-token-bridge.txt){ download="llms-token-bridge.txt" }       |
| Settlement     | Architecture, integration guides, and setup instructions for building on the Wormhole Settlement protocol | `llms-settlement.txt`     | [:octicons-copy-16:](){ .llms action="copy" data-value="llms-settlement.txt" } [:octicons-download-16:](/docs/llms-files/llms-settlement.txt){ download="llms-settlement.txt" }               |

> | Relayers       | 	Guides and reference for using Wormhole’s Relayer module, building cross-chain messaging contracts, and running custom relayers | `llms-relayers.txt`       | [:octicons-copy-16:](){ .llms action="copy" data-value="llms-relayers.txt" } [:octicons-download-16:](/docs/llms-files/llms-relayers.txt){ download="llms-relayers.txt" }                     |

Reviewer 2: ```suggestion
| Relayers       | 	Guides and reference for using Wormhole’s relayer module, building cross-chain messaging contracts, and running custom relayers | `llms-relayers.txt`       | [:octicons-copy-16:](){ .llms action="copy" data-value="llms-relayers.txt" } [:octicons-download-16:](/docs/llms-files/llms-relayers.txt){ download="llms-relayers.txt" }                     |
```

---
title: AI Resources
description: Download LLM-optimized files of the Wormhole documentation, including full content and category-specific resources for AI agents.
---

# AI Resources

> 
> ## Download LLM Files
> 
> Wormhole provides `.txt` files containing the documentation content and navigation structure, optimized for use with large language models (LLMs) and AI tools. These resources help build AI assistants, power code search, or enable custom tooling trained on Wormhole’s docs.

Reviewer 2: ```suggestion
Wormhole provides `.txt` files containing the documentation content and navigation structure, optimized for use with large language models (LLMs) and AI tools. These resources help build AI assistants, power code search, or enable custom tooling trained on Wormhole’s docs.

## Download LLM Files
```

| Category       | Description       | File  | Actions  |
|----------------|-------------------|-------|----------|
| Index         | Navigation index of all Wormhole documentation pages | `llms.txt`         | [:octicons-copy-16:](){ .llms action="copy" data-value="llms.txt" } [:octicons-download-16:](/docs/llms-files/llms.txt){ download="llms.txt" }                           |
| Full Documentation  | Full content of all documentation pages | `llms-full.txt`         | [:octicons-copy-16:](){ .llms action="copy" data-value="llms-full.txt" } [:octicons-download-16:](/docs/llms-files/llms-full.txt){ download="llms-full.txt" }                           |

> | Basics         | Core concepts covering Wormhole’s architecture, guardian network, message flow, and Verifiable Action Approvals (VAAs) | `llms-basics.txt`         | [:octicons-copy-16:](){ .llms action="copy" data-value="llms-basics.txt" } [:octicons-download-16:](/docs/llms-files/llms-basics.txt){ download="llms-basics.txt" }                           |

Reviewer 2: ```suggestion
| Basics         | Wormhole's architecture, security, and core components to help understand how the protocol works | `llms-basics.txt`         | [:octicons-copy-16:](){ .llms action="copy" data-value="llms-basics.txt" } [:octicons-download-16:](/docs/llms-files/llms-basics.txt){ download="llms-basics.txt" }                           |
```

> | Reference      | Reference details such as chain IDs, contract addresses, and finality rules | `llms-reference.txt`      | [:octicons-copy-16:](){ .llms action="copy" data-value="llms-reference.txt"} [:octicons-download-16:](/docs/llms-files/llms-reference.txt){ download="llms-reference.txt" }                   |

Reviewer 2: ```suggestion
| Reference      | Reference details such as chain IDs, contract addresses, and finality levels | `llms-reference.txt`      | [:octicons-copy-16:](){ .llms action="copy" data-value="llms-reference.txt"} [:octicons-download-16:](/docs/llms-files/llms-reference.txt){ download="llms-reference.txt" }                   |
```

| MultiGov       | Governance system using Wormhole for seamless voting and proposal execution across multiple blockchain networks | `llms-multigov.txt`       | [:octicons-copy-16:](){ .llms action="copy" data-value="llms-multigov.txt" } [:octicons-download-16:](/docs/llms-files/llms-multigov.txt){ download="llms-multigov.txt" }                     |
| Queries        | Fetch on-chain data from multiple networks to fuel analytics and insights | `llms-queries.txt`        | [:octicons-copy-16:](){ .llms action="copy" data-value="llms-queries.txt" } [:octicons-download-16:](/docs/llms-files/llms-queries.txt){ download="llms-queries.txt" }                        |
| Solidity SDK   | Smart contract tools and examples for integrating Wormhole directly into Solidity projects | `llms-solidity-sdk.txt`   | [:octicons-copy-16:](){ .llms action="copy" data-value="llms-solidity-sdk.txt" } [:octicons-download-16:](/docs/llms-files/llms-solidity-sdk.txt){ download="llms-solidity-sdk.txt" }         |

> | TypeScript SDK | Transform multichain development with a toolkit that minimizes complexity | `llms-typescript-sdk.txt` | [:octicons-copy-16:](){ .llms action="copy" data-value="llms-typescript-sdk.txt" }   [:octicons-download-16:](/docs/llms-files/llms-typescript-sdk.txt){ download="llms-typescript-sdk.txt" } |

Reviewer 2: I'm not sure about these descriptions. Some of them sound like marketing materials. I also feel like in the first four, we're describing what's in the llms files, then we start describing products
